README.md

```diff
@@ -192,10 +194,44 @@ of all services that are accessible from your local machine
 Makes sure kubectl is installed and available for use. Customize the version
 by specifying the `KUBECTL_VERSION` envrionmental variable. Default: `v1.3.6`.
 
+
 ## Assumptions
 
 * In your Deployment file, specify imagePullPolicy: IfNotPresent
 
+# Errata
```

Comment:
`Errata` may not be the correct word here, this doesn't appear to a correction. 

This section seems like it should be included as docs for the deploy script based on the current implementation.